### PR TITLE
adding systemd unit which boots um link-lab vm

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: modules/management/default.nix [hunk 1]
```diff
@@ -1,6 +1,4 @@
 {
   imports = [
-    ./bookstack.nix
```
Comment (MarcelCoding) [inline modules/management/default.nix:3]: Don't kill our book stack

File: modules/management/link-lab-vm.nix [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        -device virtio-vga-gl -display sdl,gl=on \
+        -device intel-hda -device hda-duplex \
+        -device virtio-serial -chardev spicevmc,id=vdagent,debug=0,name=vdagent \
+        -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 \
+        -hda $FILE \
+        -cdrom /var/lib/link-lab-vm/ubuntu.iso \
+        -netdev bridge,id=eth0,br=iqceth0 \
+        -device e1000,netdev=eth0,mac=52:54:00:05:06:10,bus=pci.0,addr=0x2,id=net0 \
+        -netdev bridge,id=esa0,br=iqcbr0 \
+        -device virtio-net-pci,netdev=esa0,mq=on,vectors=6,bus=pci.0,addr=0x3,id=net1
```
Comment (MarcelCoding) [inline modules/management/link-lab-vm.nix:60]: The networking config does not seem to be correct

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

File: modules/management/default.nix (hunk 1 above)
Comment (tanneberger) [reply modules/management/default.nix:3]: why is it included there?

Comment (MarcelCoding) [reply modules/management/default.nix:3]: Because bookstack is migrated into its own vm yet.

File: flake.nix [hunk 3]
```diff
@@ -292,19 +292,33 @@
...
           modules = [
-            ifstate.nixosModules.default
+            ifstate.nixosmodules.default
             { nixpkgs.overlays = [ ifstate.overlays.default ]; }
-            microvm.nixosModules.microvm
-            sops-nix.nixosModules.default
+            microvm.nixosmodules.microvm
+            sops-nix.nixosmodules.default
             ./hosts/ixp-as11201
             ./modules/dd-ix
```
Comment (MarcelCoding) [inline flake.nix:304]: Why remove the casing? This surely breaks this

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Comment by MarcelCoding** [comment]
vm already running

**Pull request closed without merging**